go/oasis-node/cmd/debug/txsource/workload/queries.go

```diff
@@ -416,6 +439,11 @@ func (q *queries) Run(gracefulExit context.Context, rng *rand.Rand, conn *grpc.C
 		)
 		return fmt.Errorf("Runtime unmarshal: %w", err)
 	}
+	rtState, ok := doc.RootHash.RuntimeStates[q.runtimeID]
```

Comment:
You could also use the `GetGenesisBlock` query on the roothash backend. Maybe we should expose this via the runtime client as well.

Reply:
:+1: yeah, i can do that

Reply:
Done, added `GetGenesisBlock` client method, please check again